languages/elixir/exercises/concept/charlists/.meta/design.md

```diff
@@ -0,0 +1,43 @@
+## Learning objectives
+
+After completing this exercise, the student should:
+
+- Know what charlists are and when they are used.
+- Know what chars are (code points).
+- Know how to use `?` to get a character's code point.
+- Know how to use `case` to match a given expression against the given clauses.
+
+## Out of scope
+
+- Changing charlists to strings and strings to charlists.
+- Pin operator.
+- Ranges.
+
+## Prerequisites
+
+This exercise's prerequisites are:
```

Comment:
I've removed this introduction text from the C# and F# exercise (see [this PR](https://github.com/exercism/v3/commit/47c3688140c6b2cf8834452d2a9f60f5e72293e2#diff-b7db3853b2fc86fcaf190b3447a65b45L17)), as it doesn't add much value. Feel free to remove it if you like.